### DISCUSSION TITLE
OGC API Coverages Code Sprint

### DISCUSSION THREAD

**Discussion by doublebyte1** [geopython/pygeoapi#1536] [General]
The [OGC API Coverages Code Sprint](https://github.com/opengeospatial/ogcapi-coverages/wiki/February-2024-OGC-API-%E2%80%90-Coverages-Virtual-Code-Sprint) is running online: until the 15th of February.
They are doing technology integration experiments using pygeoapi, so it would be great to have someone with knowledge about the implementation stepping in @Alex-NRCan @tomkralidis ?

**Comment by tomkralidis** [comment]
Good to hear! I can be available online tomorrow and am on Discord for any questions.

**Reply by doublebyte1** [reply]
@tomkralidis the final demo is tomorrow at 17 EDT
